Packs/Base/ReleaseNotes/1_32_28.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CommonServerPython
+Added the ability to custom the http requests query params encoding.
```

Comment:
```suggestion
Added the ability to customize the http requests query params encoding.
```

I just want to make sure that it is the **query** params and not the **quote** params.